tests/test_hessian.py

```diff
@@ -166,6 +168,44 @@ def test_hessian_freqs():
     assert np.isclose(nu_3, 3651.462209, atol=1.0)
 
 
+def test_hessian_scaled_freqs():
+
+    h2o = Molecule(smiles='O')
+    h2o.hessian = h2o_hessian_arr
+
+    nu_no_scaling = h2o.hessian.frequencies_proj[-1]
+
+    Config.freq_scale_factor = 0.9
+    h2o.hessian = h2o_hessian_arr
+
+    assert np.isclose(0.9 * nu_no_scaling,
+                      h2o.hessian.frequencies_proj[-1]
+                      )
+
+    Config.freq_scale_factor = None
+
+
+def test_hessian_scale_factor():
+
+    h2o = Molecule(smiles='O')
+    hessian = Hessian(h2o_hessian_arr, atoms=h2o.atoms, functional=pbe0)
+
+    assert np.isclose(hessian._freq_scale_factor, 0.96)
```

Comment:
Maybe a quick comment to refer to where these magic numbers have come from (I know they are referred in the main code tho)